composer.json

```diff
@@ -32,7 +32,8 @@
 		"bower-asset/ng-wig" : "~3.0.0",
 		"bower-asset/twigjs-bower": "~0.10.5",
 		"bower-asset/angular-filter" : "~0.5.15",
-		"bower-asset/bowser": "~1.7.3"
+		"bower-asset/bowser": "~1.7.3",
+		"bower-asset/echarts": "*"
```

Comment:
You have to provide a version with a constraint, otherwise its not backward save.  latest seems to 3.8.5 so `~3.8.5` would be a good constraint.